packages/commerce-sdk-react/src/provider.tsx

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright (c) 2022, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import {ReactElement} from 'react'
+
+interface CommerceAPIProviderProps {
+    clientId: string
```

Comment:
Minor: need semicolons to satisfy eslint
```suggestion
    clientId: string;
```

There are also other files with similar interface needing semicolons

Reply:
Update: actually, maybe there's something wrong with my eslint? 🤔

Reply:
Update 2: recently merged into this PR is #671, where eslint now understands typescript code.